Description: Upstream changes introduced in version 1.50-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 fio (1.50-1) unstable; urgency=low
 .
   * Thanks for importing, Ritesh. Found my mistake.
   * New upstream version 1.5.0 (Closes: #613432).
   * Updated to Standards-Version to 3.9.1, no changes needed.
 .
 The person named in the Author field signed this changelog entry.
Author: Martin Steigerwald <ms@teamix.de>
Bug-Debian: http://bugs.debian.org/613432

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- fio-1.50.orig/Makefile
+++ fio-1.50/Makefile
@@ -58,9 +58,9 @@ endif
 endif
 
 INSTALL = install
-prefix = /usr/local
+prefix = /usr
 bindir = $(prefix)/bin
-mandir = $(prefix)/man
+mandir = $(prefix)/share/man
 
 .c.o:
 	$(QUIET_CC)$(CC) -o $@ -c $(CFLAGS) $(CPPFLAGS) $<
